### DISCUSSION TITLE
What is the correct way to access commit diff results via http?

### DISCUSSION THREAD

**Discussion by madgetr** [huggingface/huggingface.js#608] [Q&A] [answered]
Data I am interested in:
![image](https://github.com/huggingface/huggingface.js/assets/16808224/cada880a-bc46-496b-869b-02adb083b6a7)
Here's the endpoint to list commits
https://huggingface.co/api/models/SimonMA/Codellama-7b-lora-rps-adapter/commits/main

**Comment by coyotte508** [comment, marked as answer]
https://huggingface.co/SimonMA/Codellama-7b-lora-rps-adapter/commit/739dded9ac6447737fe2062acdc19a109633427a.diff

**Reply by madgetr** [reply]
Thanks, is there args such as name-only?

**Reply by coyotte508** [reply]
you also have https://huggingface.co/api/models/SimonMA/Codellama-7b-lora-rps-adapter/compare/main..739dded9ac6447737fe2062acdc19a109633427a

you can get a specific file https://huggingface.co/SimonMA/Codellama-7b-lora-rps-adapter/commit/739dded9ac6447737fe2062acdc19a109633427a.diff?file=adapter_model.safetensors

no other endpoints (to get just list of files changed), you need to parse the diff (git format)
